packages/atomic/src/components/searchbox/base-search.tsx

```diff
@@ -0,0 +1,182 @@
+import {Component, Event, EventEmitter, h, Prop} from '@stencil/core';
+import {I18nState} from '../../utils/initialization-utils';
+import {Combobox} from '../../utils/combobox';
+
+import ClearIcon from 'coveo-styleguide/resources/icons/svg/clear.svg';
+import SearchIcon from 'coveo-styleguide/resources/icons/svg/search.svg';
+
+@Component({
+  tag: 'base-search',
```

Comment:
Yeah just to reiterate what I stated in stand-up, this can probably be replaced with a functional component but it'll take some refactoring in how we handle the combobox class. So for now we're going to keep it as an internal full fledged web component, in order to start writing tests and let Lucille/William start testing it.